From 62bda7ae7bf0880201c4872c54e5b530b2fec27b Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@xilinx.com>
Date: Mon, 16 Dec 2013 16:37:32 +1000
Subject: [PATCH 37/43] microblaze: Add build_gdbserver=yes to top level
 configure.tgt

For Microblaze linux toolchains, set the build_gdbserver=yes
to allow driving gdbserver configuration from the upper level

This patch has been absorbed into the original patch to add
linux gdbserver support for Microblaze.

Signed-off-by: David Holsgrove <david.holsgrove@xilinx.com>
---
 gdb/configure.tgt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gdb/configure.tgt b/gdb/configure.tgt
index 622bd486b3..989523735b 100644
--- a/gdb/configure.tgt
+++ b/gdb/configure.tgt
@@ -405,6 +405,7 @@ microblaze*-*-*)
 	# Target: Xilinx MicroBlaze running standalone
 	gdb_target_obs="microblaze-tdep.o"
 	gdb_sim=../sim/microblaze/libsim.a
+	build_gdbserver=yes
 	;;
 
 mips*-*-linux*)
-- 
2.17.1

